q2_diversity/_beta/_method.py

```diff
@@ -42,7 +43,7 @@ def non_phylogenetic_metrics():
             'correlation', 'hamming', 'jaccard', 'chebyshev', 'canberra',
             'braycurtis', 'mahalanobis', 'yule', 'matching', 'dice',
             'kulsinski', 'rogerstanimoto', 'russellrao', 'sokalmichener',
-            'sokalsneath', 'wminkowski', 'aitchison'}
+            'sokalsneath', 'wminkowski', 'aitchison', 'canberra-adkins'}
```

Comment:
Super minor, but the other metrics (really just UniFrac) seem to use `_` in their spelling.

Reply:
```suggestion
            'sokalsneath', 'wminkowski', 'aitchison', 'canberra_adkins'}
```

Reply:
:+1:, applied the suggestion